mw-chat/internal/db/query/messages.query.sql

```diff
@@ -1,9 +1,25 @@
 -- name: CreateMessage :one
 INSERT INTO messages (owner_uuid, room_uuid, text)
 VALUES (@owner_uuid, @room_uuid, @text)
-RETURNING owner_uuid, text;
+RETURNING uuid, owner_uuid, text;
 
 -- name: GetMessagesByRoomUUID :many
-SELECT owner_uuid, text 
+SELECT
+    messages.owner_uuid,
+    messages.text,
+    ARRAY(
+    SELECT receiver_uuid
+     FROM message_status
+     WHERE messages.uuid = message_status.message_uuid
+        AND is_read = true
+     ORDER BY updated_at DESC
+    )::UUID[] AS message_status_user_uuids,
+    ARRAY(
+    SELECT updated_at
+     FROM message_status
+     WHERE messages.uuid = message_status.message_uuid
+        AND is_read = true
+     ORDER BY updated_at DESC
+    )::UUID[] AS message_status_updated_at
```

Comment:
lets add limit  100 for any message response otherwise request will be too heavy for each user soon)
in future we will add pagination for messages